frontend/packages/kubevirt-plugin/src/selectors/vm/selectors.ts

```diff
@@ -55,6 +53,52 @@ export const getVolumes = (vm: VMKind, defaultValue: V1Volume[] = []): V1Volume[
 export const getDataVolumeTemplates = (vm: VMKind, defaultValue = []) =>
   _.get(vm, 'spec.dataVolumeTemplates') == null ? defaultValue : vm.spec.dataVolumeTemplates;
 
+export const getConfigMapVolumes = (vm: VMKind, defaultValue: V1Volume[] = []): V1Volume[] =>
+  getVolumes(vm, defaultValue).filter((vol) => Object.keys(vol).includes('configMap'));
```

Comment:
why do we have so many functions copy pasted functions? Can't we just do one and filter it by VolumeType?